presto-docs/src/main/sphinx/release/release-0.261.rst

```diff
@@ -0,0 +1,45 @@
+=============
+Release 0.261
+=============
+
+**Highlights**
+==============
+
+**Details**
+===========
+
+General Changes
+_______________
+* Fix Aggregation IF to FILTER by not unwrapping the IF expression instead Aggregation.
+* Fix marking coordinator active so it is only done when the required minimum resource managers and coordiantors are available in a cluster.
```

Comment:
@swapsmagic  What is this minimum? Is it configurable? Can you add a link to the documentation for this new configuration?

Reply:
* Fix marking coordinator active so it is only done when the required minimum resource managers and required minimum coordinators are available in a cluster. Required resource managers can be configured by ``cluster.required-resource-managers-active`` (default value: 1, disaggregated coordinator only). Required coordinators can be configured by ``cluster.required-coordinators-active`` (default value: 1)